### DISCUSSION TITLE
[Documentation Request] Environment Management

### DISCUSSION THREAD

**Discussion by KarinTreyer** [brightway-lca/brightway-documentation#126] [Documentation Request]
### Request Category

Creation of new page

### Page Title

FAQ/Environment Management

### Description

Saving information of an environment, and restoring it easily at any time.

### Can you help?

- [X] I could help with the content of this documentation request.
- [ ] I could help with the technical implementation of this documentation request.

**Comment by michaelweinold** [comment]
If you _have an existing environment and would like to re-create it at a later time_:

```
conda list -n <ENVIRONMENT_NAME> --export > ./environment.yml
```

and then recreate it with:

```
conda env create -f environment.yml
```

**Comment by michaelweinold** [comment]
Implemented with https://github.com/brightway-lca/brightway-documentation/commit/6a13fcf8aa1c9d208c13ce0922dc1e499dad22d5. Thank you for suggesting this, @KarinTreyer!
